pydy/codegen/tests/test_c_code.py

```diff
@@ -25,7 +25,7 @@ def setup(self):
         self.arguments = (list(sm.ordered(sys.constants_symbols)),
                           sys.coordinates,
                           sys.speeds,
-                          sys.specifieds_symbols)
+                          list(sm.ordered(sys.specifieds_symbols)))
```

Comment:
Yes, this seems correct. The functions in System that extract the constants, coordinates, speeds, and specifieds can extract them in different orders depending on the version of SymPy, the OS, etc. In general, extracting symbols from SymPy expressions does not guarantee some order.